src/Analyser/ResultCache/ResultCacheManager.php

```diff
@@ -285,8 +288,10 @@ public function restore(array $allAnalysedFiles, bool $debug, bool $onlyFiles, ?
 		$filesToAnalyseCount = count($filesToAnalyse);
 
 		if ($output->isDebug()) {
+			$elapsed = microtime(true) - $startTime;
 			$output->writeLineFormatted(sprintf(
-				'Result cache restored. %d %s will be reanalysed.',
+				'Result cache restored in %f seconds. %d %s will be reanalysed.',
```

Comment:
Please only output the variant with seconds if it took more than 5 seconds. Otherwise it's distracting, thanks.